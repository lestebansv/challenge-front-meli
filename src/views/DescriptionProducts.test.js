import { render } from "@testing-library/react";
import connection from "../utils/channel";
import DescriptionProduct from "./DescriptionProduct";

describe("DescriptionProduct", () => {

    it('validate render view', async () => {
        await connection('MLA1135260991', "product");

        const { container } = render(<DescriptionProduct />);

        expect(container).toMatchSnapshot();
    });
    
});