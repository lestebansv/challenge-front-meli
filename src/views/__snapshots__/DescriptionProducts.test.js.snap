// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DescriptionProduct validate render view 1`] = `<div />`;
